app/scripts/filters/exists.coffee

```diff
@@ -17,6 +17,8 @@ angular.module('edudashAppFil')
                 for prop, value of input
                     return true;
                 return false
+            if angular.isNumber input && input == NaN
```

Comment:
The unit tests only pass because they are incomplete :disappointed:

so, first add this regression test which (pretty sure) should fail:

```coffeescript
   it 'should be true for zero', ->
     zero = 0
     expect(exists zero).toBe true
```

then change

```coffeescript
if angular.isNumber input && input == NaN
  return false
```
to
```coffeescript
if angular.isNumber input
  return input == input  # false for NaN
```

or something

I think

Good luck :wink: